tuned.spec

```diff
@@ -60,6 +61,10 @@ Requires(preun): systemd
 Requires(postun): systemd
 BuildRequires: %{_py}, %{_py}-devel
 # BuildRequires for 'make test'
+# python-mock is needed for python-2.7, but it's not available on RHEL-7
+%if %{without python3} && 0%{?rhel} < 8
+BuildRequires: %{_py}-mock
```

Comment:
I don't get it. Why are you requiring python-mock on RHEL-7 if it's not available there?

Reply:
It should be fixed now.